airstack-modules/modules/airstack/domain-name/domain-name.ts

```diff
@@ -1102,12 +1042,12 @@ export namespace domain {
    */
   function recurseSubdomainCountDecrement(domain: AirDomain, chainId: string, block: AirBlock, tokenAddress: string): string | null {
```

Comment:
Please verify if the return values are needed? 
I dont see its being used.

If `domain.id` is needed, can you check which `id` should should be returned? Its a recursive call.

Reply:
removed return statements.

Reply:
done